pyproject.toml

```diff
@@ -149,6 +156,8 @@ sphinx = "*"
 recommonmark = "*"
 sphinx-rtd-theme = "*"
 pygments-github-lexers = "*"
+flake8 = "*"
+pre-commit = "*"
```

Comment:
Why it's here?

Reply:
It is in dev sector so that **developers could have the tool when they `poetry install` at once.** I'm not sure if this really a preferred way of developing the lib. Because I see in the test that the install is done by `install.py`

Where should it be? Are you install these tools in global separately? I will delete those 2 lines for you anyway :)